tests/test_nonblocking_stream.py

```diff
@@ -80,6 +81,8 @@ def setup_server(self):
             'streamtest' + str(random.randint(100000, 999999)))
         self.log_file = os.path.join(self.log_directory, 'streamtest')
         os.makedirs(self.log_directory)
+        process_env = os.environ.copy()
+        process_env['TRITON_ZMQ_PORT'] = str(TEST_TRITON_ZMQ_PORT)
```

Comment:
Sort of. Tests were failing to run if tritond was already running. This just lets the tests run safely on the non-default port, no matter if tritond is running on the system or not.